hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2976,6 +2974,24 @@ public boolean supportsNoWait() {
 		return false;
 	}
 
+	/**
+	 * Does this dialect/database support LO_CREATE and related functions.
+	 *
+	 * @return {@code true} if LO_CREATE and related functions are supported
+	 */
+	public boolean supportsLoFunctions() {
+		return false;
+	}
+
+	/**
+	 * Does this dialect/database support mixed type arithmetic (e.g. int + float).
+	 *
+	 * @return {@code true} if mixed type arithmetic is supported
+	 */
+	public boolean supportsMixedTypeArithmetic() {
+		return true;
+	}
+
```

Comment:
can we avoid the introduction of these 2 methods and instead use `@SkipForDialect` annotation?

Reply:
Sounds good. I've removed these new methods